src/main/java/org/kizombadev/markdownparser/entities/Syntax.java

```diff
@@ -83,7 +82,7 @@ public SyntaxType getType() {
         return type;
     }
 
-    @Override
+    /*@Override
     public boolean equals(Object o) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)